docs/ui-ux-feedback.md

```
@@ -0,0 +1,183 @@
+# Forgather Admin UI/UX 피드백 리포트
+
+## 분석 대상
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)